edit/networking.k8s.io.ingress/index.vue

```diff
@@ -163,11 +177,7 @@ export default {
 
       this.$set(this.value, 'spec', this.spec);
```

Comment:
(I know this isn't new code but) this pattern of cloning metadata/spec in data and then setting them back on save is weird... the model object (for anything going through ResourceDetail) is already a clone of the original model being edited, and the original is available as originalValue if needed.

Reply:
Removed the cloning.